### DISCUSSION TITLE
Can capacitor be used as a container in an existing app?

### DISCUSSION THREAD

**Discussion by leekeal** [ionic-team/capacitor#4136] [Q&A]
Can capacitor be used as a container in an existing app? Has anyone tried it?

**Comment by Aarbel** [comment]
@leekeal yes, upvote my question here:

https://github.com/ionic-team/capacitor/discussions/4150

For the moment the "only" limit seems to access plugins from this external app url (which is really necessary to achieve the goal of simple Progressive Web App with Capacitor)

**Comment by Aarbel** [comment]
@leekeal, after many tests, yes Capacitor can wrap and external SPA or PWA url. The capacitor plugins manage all the compatibility will all environments (browsers, native wkwebviews, chromium), really sweet.
